mgmt_cli_test.py

```diff
@@ -58,8 +58,7 @@ def install_awscli_dependencies(self, node, destination):
         cmd = f'''sudo yum install -y epel-release
                   sudo yum install -y python-pip
                   sudo yum remove -y epel-release
-                  sudo pip install awscli
-                  sudo pip install boto3
```

Comment:
don't we need `boto3` anymore?

Reply:
it's going to be installed as part of awscli, the fact we installed it might be part of the issue.